tmt/steps/discover/fmf.py

```diff
@@ -301,6 +301,27 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
             if not url:
                 assert_git_url(self.step.plan.name)
 
+        # for imported plans print details about a remote plan
+        parent = cast(Optional[tmt.steps.discover.Discover], self.parent)
+        if parent and parent.plan._original_plan and \
+                parent.plan._original_plan._remote_plan_fmf_id:
+
+            remote_plan_id = parent.plan._original_plan._remote_plan_fmf_id
+
+            # FIXME: cast() - https://github.com/python/mypy/issues/7981
+            # Note the missing Optional for values - to_minimal_dict() would
+            # not include unset keys, therefore all values should be valid.
+            for key, value in cast(Dict[str, str], remote_plan_id.to_minimal_spec()).items():
+
+                # Will be printed out as "ref" instead
+                if key == '_ref_expanded':
```

Comment:
A leftover, deserves the same fate it encountered in tmt/steps/discover/shell.py you fixed.

Reply:
Ouch. Thank you. Removed.